CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 * Require dart_style >= 2.3.7, so that the current Dart language version can be
   passed to `DartFormatter`.
 * Add topics to `pubspec.yaml`.
+* Fix a bug where typedef-aliases in type arguments were not correctly
```

Comment:
I think instead of "typedef-aliases," we generally call them "type aliases."

Reply:
Oh yeah, where do you think I got it from? 😉😜
https://github.com/dart-lang/mockito/blob/f72791d3dd5cb2acc4f5c7da065ffe70d2ad1e0b/CHANGELOG.md?plain=1#L51-L60

I'll fix it anyway 😂